doc/wallet-passphrase.md

```diff
@@ -0,0 +1,21 @@
+## Wallet Passphrase
+
+Understanding the nuances of wallet security is essential to storing your Bitcoin safely. One crucial aspect of safeguarding funds is the utilization of a wallet passphrase or password, implemented through wallet encryption introduced in Bitcoin Core [v0.4.0](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.4.0.md). Let's explore key points to demystify the nature of a wallet passphrase and the encryption process, emphasizing what they do and what they don't do:
+
+  - **Not the Seed:**
+The wallet passphrase is distinct from the seed phrase (recovery phrase). The seed phrase is a crucial backup that can be used to restore access to your wallet and recover your funds. The passphrase, on the other hand, is an additional layer of security specific to secure the private keys in the wallet. Anybody who discovers a wallet's seed phrase can steal all the bitcoins if the seed isn't also protected by a secret passphrase.
```

Comment:
What do you mean with seed phrase/recovery phrase? I think you're confused with BIP39, which is not in Bitcoin Core